[2024-02-26T15:52:12.521+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2024-02-26T15:52:09+00:00 [queued]>
[2024-02-26T15:52:12.536+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2024-02-26T15:52:09+00:00 [queued]>
[2024-02-26T15:52:12.539+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-02-26T15:52:12.555+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): runme_2> on 2024-02-26 15:52:09+00:00
[2024-02-26T15:52:12.561+0000] {standard_task_runner.py:60} INFO - Started process 2187 to run task
[2024-02-26T15:52:12.576+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2024-02-26T15:52:09+00:00', '--job-id', '2204', '--raw', '--subdir', '/home/***/.local/lib/python3.11/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp58eleel6']
[2024-02-26T15:52:12.584+0000] {standard_task_runner.py:88} INFO - Job 2204: Subtask runme_2
[2024-02-26T15:52:12.686+0000] {task_command.py:423} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2024-02-26T15:52:09+00:00 [running]> on host d07e1489670f
[2024-02-26T15:52:12.819+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2024-02-26T15:52:09+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-26T15:52:09+00:00'
[2024-02-26T15:52:12.822+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-26T15:52:12.824+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20240226" && sleep 1']
[2024-02-26T15:52:12.842+0000] {subprocess.py:86} INFO - Output:
[2024-02-26T15:52:12.844+0000] {subprocess.py:93} INFO - example_bash_operator__runme_2__20240226
[2024-02-26T15:52:13.848+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-26T15:52:13.908+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20240226T155209, start_date=20240226T155212, end_date=20240226T155213
[2024-02-26T15:52:13.956+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-26T15:52:13.978+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
